src/components/users/userProfile/UserProfile.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import client from "../../../utils/client";
+import { useParams } from "react-router-dom";
+
+const UserProfile = () => {
+  const initialProfileState = {
+    first_name: "",
+    last_name: "",
+    email: "",
+    biography: "",
+    github_url: "",
+  };
+  const [profile, setProfile] = useState(initialProfileState);
+  const { id } = useParams();
+
+  useEffect(() => {
+      foundProfile()
+  }, [])
+
+  const foundProfile = () => {
```

Comment:
maybe a more intuative name could be used?

Reply:
I have changed the function name to handleFindProfile, hopefully this is more suitable!